index.html

```diff
@@ -0,0 +1,231 @@
+        
+                if( n1 >= n2 && n1 >= n3)
+                    System.out.println(n1 + " is the largest number.");
+        
+                else if (n2 >= n1 && n2 >= n3)
+                    System.out.println(n2 + " is the largest number.");
+        
+                else
+                    System.out.println(n3 + " is the largest number.");
+            }
+        }
+        </code>
+      </section>
+
+      <section class="main-section">
+        <header id="syntax">
+          <h1>Syntax</h1>
+        </header>
+        <p>
+          Variables in JavaScript can be defined using either the var,[69] let[70] or const[71] keywords. Variables
+          defined
+          without keywords will be defined at the global scope.
+        </p>
+        <code>
+          <p>
+            // Declares a function-scoped variable named `x`, and implicitly assigns the
+            // special value `undefined` to it. Variables without value are automatically
+            // set to undefined.
+            var x;
+          </p>
+          <p>
+            // Variables can be manually set to `undefined` like so
+            var x2 = undefined;
+          </p>
+          <p>
+            // The keyword `const` means constant, hence the variable cannot be reassigned
+            // as the value is `constant`.
+            const z = "this value cannot be reassigned!";
+          </p>
+        </code>
+      </section>
+
+      <section class="main-section">
+        <header id="development_tools">
+          <h1>Development Tools</h1>
+        </header>
+        Important tools have evolved with the language. <br>
+        <ul>
+          <li> Every major web browser has built-in web development tools, including a JavaScript debugger. </li>
+          <li> Static program analysis tools, such as ESLint and JSLint, scan JavaScript code for conformance to a set
+            of
+            standards and guidelines. </li>
+          <li> Some browsers have built-in profilers. Stand-alone profiling libraries have also been created, such as
+            benchmark.js and jsbench. </li>
+          <li> Many text editors have syntax highlighting support for JavaScript code. </li>
+        </ul>
+        <code>
+          public class Largest {
+  
+            public static void main(String[] args) {
+        
+                double n1 = -4.5, n2 = 3.9, n3 = 5.5;
+        
+                if(n1 >= n2) {
+                    if(n1 >= n3)
+                        System.out.println(n1 + " is the largest number.");
+                    else
+                        System.out.println(n3 + " is the largest number.");
+                } else {
+                    if(n2 >= n3)
+                        System.out.println(n2 + " is the largest number.");
+                    else
+                        System.out.println(n3 + " is the largest number.");
+                }
+            }
+        }
+        </code>
+      </section>
+    </div>
```

Comment:
Use better semantics. Let the div element be a section element for readable and cleaner code.